### DISCUSSION TITLE
Change the way Gradle find code signing configuration

### DISCUSSION THREAD

**Discussion by ghost** [Exodus-Privacy/exodus-android-app#126] [Ideas]
Hello,

For now, If we want to create a new release, we need to `Generate  a signed Bundle / APK...` give the path to the keystore, the keystore password, the keyAlias and the KeyPassword.

In gradle we can use variable and secret file (which will not be committed) id order to simplify this generation.

see [Pilldroid build.gradle file](https://github.com/jfoucry/Pilldroid/blob/develop/app/build.gradle).
or the [documentation](https://developer.android.com/studio/build/build-variants#signing)

**Comment by Jean-BaptisteC** [comment]
Maybe submit this new feature with next version in Kotlin
This change has an impact in Workflow Github to build app ?

**Comment by Jean-BaptisteC** [comment]
code signing config was rework with 3.1.0 version
